diff --git a/es/Image/components/usePreviewGroup.mjs b/es/Image/components/usePreviewGroup.mjs
index 0000000..1111111 100644
--- a/es/Image/components/usePreviewGroup.mjs
+++ b/es/Image/components/usePreviewGroup.mjs
@@ -12,9 +12,10 @@ const usePreview = (props) => {
 	return useMemo(() => {
 		if (props === false) return props;
-		const { onVisibleChange, onOpenChange, minScale = .32, maxScale = 32, toolbarAddon, rootClassName, imageRender, toolbarRender, ...rest } = props === true ? {} : props || {};
+		const { onVisibleChange, onOpenChange, minScale = .32, maxScale = 32, toolbarAddon, rootClassName, classNames, imageRender, toolbarRender, ...rest } = props === true ? {} : props || {};
+		const previewRootCls = cx(styles.preview, rootClassName);
 		return {
 			actionsRender: (_, info) => {
 				const originalNode = /* @__PURE__ */ jsx(Toolbar_default, {
@@ -42,7 +43,7 @@ const usePreview = (props) => {
 				onVisibleChange?.(open, !open, info.current);
 			},
-			rootClassName: cx(styles.preview, rootClassName),
+			classNames: { ...classNames, root: cx(previewRootCls, classNames?.root) },
 			...rest
 		};
 	}, [
